speedscale/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "display_name": "Speedscale",
+  "maintainer": "support@speedscale.com",
+  "manifest_version": "1.0.0",
+  "name": "speedscale",
+  "metric_prefix": "speedscale.",
+  "metric_to_check": "",
+  "creates_events": true,
+  "short_description": "",
+  "guid": "f2e3ed64-a756-440f-9a1f-43089536aa08",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Speedscale",
+  "categories": [
```

Comment:
Categories can be found on [this page](https://docs.datadoghq.com/integrations/) in the list at the top of the page. Feel free to add any categories that are relevant to your listing.

Reply:
Thank you, I was looking for that list. Perfect!